projects/pt1/python/torch_mlir_e2e_test/test_suite/pooling.py

```diff
@@ -776,6 +864,25 @@ def AvgPool1dStaticModule_basic(module, tu: TestUtils):
 
 # ==============================================================================
 
+class AdaptiveAvgPool1dGeneralDynamic(torch.nn.Module):
+    
+    def __init__(self):
+        super().__init__()
+        self.aap1d = torch.nn.AdaptiveAvgPool1d(7)
```

Comment:
nit: can you use named arguments to make easier to tell what the 7 means? Same comment for the other tests

Reply:
Let me know if my correction addresses this properly for the cited example. If so, I'll add the same kind of change to other adaptive pooling tests.